eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/BitcoinCoreWallet.scala

```diff
@@ -121,13 +120,15 @@ class BitcoinCoreWallet(rpcClient: BitcoinJsonRPCClient)(implicit ec: ExecutionC
   for {
     exists <- getTransaction(tx.txid).map(_ => true).recover { case _ => false }
     doublespent <- if (exists) {
-      // if the tx is in the blockchain, it can't have been doublespent
+      // if the tx is in the blockchain, it can't have been double-spent
```

Comment:
If `getTransaction` fails we should explicitly check the returned error which should be `No such mempool or blockchain transaction. Use gettransaction for wallet transactions. (code: -5)`  and should not contain `Blockchain transactions are still in the process of being indexed` (we cannot use error codes to differentiate here, it would also be -5)

Reply:
Fixed with cc8d7fb